src/handling/world/CharacterTransfer.java

```diff
@@ -434,7 +430,12 @@ public void readExternal(final ObjectInput in) throws IOException, ClassNotFound
         for (int i = 0; i < rebsize; i++) {
             this.rebuy.add((Item) in.readObject());
         }
-
+		
+        final int favoritesize = in.readShort();
+        for (int i = 0; i < favoritesize; i++) {
```

Comment:
It was more so a la copy paste for CharacterTransfer, stuck with it for the consistency.